sig-on-premise/OWNERS

```diff
@@ -1,8 +1,6 @@
 reviewers:
-  - zen
-  - marcoceppi
-  - dghubble
+  - sig-on-premise-leads
 approvers:
-  - zen
-  - marcoceppi
-  - dghubble
+  - sig-on-premise-leads
+labels:
+  - sig/onprem
```

Comment:
Hope the names can be consistent one day... onprem vs on-premise
ref: https://github.com/kubernetes/community/issues/703

Reply:
I hope they can to. This isn't to change that though -- just capturing the current state of the world.